src/Commands/CommandFactory.cs

```diff
@@ -285,13 +285,17 @@ private void RegisterAppConfigCommands()
         var keyValue = new CommandGroup("kv", "App Configuration key-value setting operations - Commands for managing complete configuration settings including values, labels, and metadata");
         appConfig.AddSubGroup(keyValue);
 
+        var featureFlag = new CommandGroup("ff", "App Configuration feature flag operations - Commands for managing feature flags");
```

Comment:
I believe the tool name is generated from this. "feature-flag"

Reply:
We had a comment above we are working on feature flags as a separate resource with their own endpoint to match the kv one. So ff works like kv, and matches the content type.

Reply:
I'm ok with ff